keps/prod-readiness/sig-storage/1495.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 1495
 alpha:
   approver: "@deads2k"
+beta:
```

Comment:
I think you can remove the alpha related lines 2 and 3.

Reply:
I don't see the harm leaving it in. Are you sure the procedure is to remove old versions?

Reply:
I don't see older versions kept in the other files: https://github.com/kubernetes/enhancements/tree/master/keps/prod-readiness/sig-storage